devmapper/devmapper.go

```diff
@@ -501,7 +521,7 @@ func (d *Driver) CreateVolume(req Request) error {
 	}
 	if backupURL == "" {
 		// format the device
-		if _, err := util.Execute("mkfs", []string{"-t", "ext4", dev}); err != nil {
+		if err := d.createFilesystem(dev); err != nil {
```

Comment:
I don't understand your question well. Could you be more specific about
your question or what would you suggest?